src/index.js

```diff
@@ -1,10 +1,9 @@
 import isArray from 'lodash.isarray';
 import isFunction from 'lodash.isfunction';
 import isObject from 'lodash.isobject';
-import merge from 'lodash.merge';
```

Comment:
If we no longer depend on `lodash.merge` we should remove it from the `package.json` too